fdbclient/ClusterInterface.h

```diff
@@ -119,13 +119,14 @@ struct OpenDatabaseRequest {
 	Arena arena;
 	StringRef issues, traceLogGroup;
 	VectorRef<ClientVersionRef> supportedVersions;
+	bool client_tls_configured;
 	UID knownClientInfoID;
 	ReplyPromise< struct ClientDBInfo > reply;
 
 	template <class Ar>
 	void serialize(Ar& ar) {
 		ASSERT( ar.protocolVersion() >= 0x0FDB00A400040001LL );
-		serializer(ar, issues, supportedVersions, traceLogGroup, knownClientInfoID, reply, arena);
+		serializer(ar, issues, supportedVersions, client_tls_configured, traceLogGroup, knownClientInfoID, reply, arena);
```

Comment:
You should check the protocol version like

```
if (ar.isDeserializing && ar.protocolVersion() <  X /* current Net2::currentProtocolVersion */) {
   // old serialize call
} else {
  // new serialize call
}
```

X should be taken from https://github.com/apple/foundationdb/blob/master/flow/Net2.actor.cpp#L58

Reply:
We don't need to consider multi-version protocol because this is after the connection between client and cluster has been established. 
Only the interfaces that are persisted in the database or disk (e.g., storage server interface) need to consider the multi-version serialization.